### DISCUSSION TITLE
Replace twitchemotes.com as a reference website for sub emotes

### DISCUSSION THREAD

**Discussion by LosFarmosCTL** [Chatterino/chatterino2#3126] [Ideas] [answered]
As some might have noticed twitchemotes.com has been broken ever since they turned off their API and twitch launched the helix API for getting sub emotes. For example, when you take a look at forsen‘s emotes, there are no normal T1 emotes showing up, for basically all other channels the animated emotes are missing (even if they are showing up, they are not animated).

This means that the right/secondary-click context menu on sub emotes in chatterino is completely useless right now (aside from chatterino also not having implemented twitchemotes.com‘s new URL format), since a lot of emotes are just not listed on the site anymore.

Would be great to get an idea for what the devs are planning for this. Should the link just be removed, or replaced with some other emote service, if yes, what service? Right now I personally use [raccatta.cc](https://emotes.raccatta.cc/twitch/), but that site does not have any dedicated pages for emotes, which makes it pretty much useless for current the chatterino implementation, although you could of course also change to something like „open channels emotes“ instead of the single specific emote.

I already opened an issue for it ( #3114 ), but i thought i would just drop it here as well, maybe some people have suggestions for what would be a good alternative, I would also highly appreciate any feedback the devs could give on this.

**Comment by pajlada** [comment, marked as answer]
Since there's no direct replacement, we've gone ahead and just removed the right-click option entirely.

https://github.com/Chatterino/chatterino2/pull/3136
